src/components/profile/statistics/Statistics.tsx

```diff
@@ -1,25 +1,17 @@
 import React, { useContext } from "react";
 import { i18nContext } from "../../../utils/i18nContext";
-import { useDarkMode } from "../../../utils/darkMode";
 import { BadgeSubojectivesCategoryCompletion } from "./badges";
 
 export const Statistics = ({ userAchievements, skillsDatas, myStatistics }) => {
   const { t } = useContext(i18nContext);
-  const { darkMode } = useDarkMode();
 
   const filterCountSkills = (label) =>
     skillsDatas.find((c) => c.label === label).CurrentSkillsAndDesires_aggregate
       .aggregate.count;
 
   return (
-    <div
-      className={`${
-        darkMode
-          ? "bg-dark-dark pb-4 pl-4 pr-4 mt-2 flex-col rounded-lg"
-          : "bg-light pb-4 pl-4 pr-4 mt-2 flex-col rounded-lg"
-      }`}
-    >
-      <h2 className="pb-2 pr-2 pt-6 text-2xl">
+    <div className="bg-light-dark dark:bg-dark-dark pb-4 pl-4 pr-4 my-2 p-2 flex-col rounded-lg">
```

Comment:
Seems like in the project, a lot of div like this one are used, could be great to use some styled components. The goal is that we would use exactly the same design for wrappers or buttons or any UI component. Very handful.
https://www.npmjs.com/package/tailwind-styled-components

Reply:
Should be very nice to implement it in Skillz

Reply:
I agree with you @PierreBrethes